lib/widgets/orderwidget.dart

```diff
@@ -6,7 +6,7 @@ import 'package:food_delivery_app/utils/universal_variables.dart';
 import 'foodTitleWidget.dart';
 
 class OrderWidget extends StatefulWidget {
-  Request request;
+  final Request request;
```

Comment:
any reason for keeping it final?

Reply:
yeah it is unmutable. It was showing some warning so i added final